.github/workflows/ci.yml

```diff
@@ -128,13 +128,6 @@ jobs:
           $connectString = "AccessKeyId=${{ secrets.AWS_ACCESS_KEY_ID }};SecretAccessKey=${{ secrets.AWS_SECRET_ACCESS_KEY }};Region=${{ secrets.AWS_REGION }};QueueNamePrefix=GHA-${{ matrix.storage }}-${{ github.run_id }}"
           echo "ServiceControl.AcceptanceTests.ConnectionString=$connectString" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
 
-      - name: Setup Microsoft SQL Server Data Store
```

Comment:
What about the same thing on Line 94?

Reply:
It's needed for the test running against sqlserver transport

Reply:
🤦 Carry on, pretend I'm not here.